exercises/scripts/Setup-LocalDBInstance.ps1

```diff
@@ -6,3 +6,9 @@ sqllocaldb create $instanceName
 sqllocaldb share $instanceName $instanceName
 sqllocaldb start $instanceName
 sqllocaldb info $instanceName
+
+$serverName = "(localdb)\" + $instanceName
+$userprofilepath = "$ENV:UserProfile\wsmicroservices-databases"
```

Comment:
We dropped all mentions of "microservices" from the workshop a while ago. It would be more appropriate to call this "particular-workshop-databases".

Reply:
While correct it is not entirely true, see the instance name of the local db :trollface:  :) Changed it as well in the changes pushed